site/content/api/Tenant-API.md

```diff
@@ -151,17 +151,39 @@ The JSON structure below contains example information for tenant `TEST_TENANT`.
 
 ## Trusted CA Format
 
-The table below provides an overview of the members defined for the *trusted-ca* JSON object:
+The *trusted-ca* property of the tenant (JSON) payload can contain either a single JSON object containing the trusted CA to use for validating certificates of devices belonging to the tenant.
++The tenant can also be configured with multiple trusted certificates. In this case, the *trusted-ca* property can contain an array of JSON objects, were each JSON object represents a single trusted CA. The table below provides an overview of the members defined in a JSON object contained in the *trust-ca* property:
 
 | Name                     | Mandatory  | Type          | Default Value | Description |
 | :------------------------| :--------: | :------------ | :------------ | :---------- |
 | *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
 | *cert*                   | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root X.509 certificate. |
 | *public-key*             | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
+| *not-before*             | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid before the given date. |
+| *not-after*              | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid after the given date. |
 | *algorithm*              | *no*       | *string*      | `RSA`        | The name of the public key algorithm. Supported values are `RSA` and `EC`. This property is ignored if the *cert* property is used to store a certificate. |
 
 * The *subject-dn* MUST be unique among all registered tenants.
 * Either the *cert* or the *public-key* MUST be set.
+* If the *cert* property is used to store a certificate then the *not-before*, *not-after* and *algorithm* properties are ignored.
+
+**Examples**
+
+Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT` configured with two trusted certificate authorities.
+
+~~~json
+ {
+   "tenant-id" : "TEST_TENANT",
+   "enabled" : true,
+   "trusted-ca": [ {
+           "subject-dn": "CN=ca,OU=Hono,O=Eclipse",
+           "public-key": "NOTAPUBLICKEY"
+         }, {
+           "subject-dn": "CN=ca,OU=Hono,O=Eclipse",
+           "public-key": "NOTAPUBLICKEY"
```

Comment:
it looks like you can use multiple public keys sharing the same subject DN. I agree, that this makes sense. However, the example should also define e.g. overlapping validity periods in order to illustrate a meaningful use case for this.
Also, we should explicitly state that a subject DN still needs to be unique across tenants. Otherwise we could not resolve a tenant based on the subject DN anymore...